NachoSpigot-API/src/main/java/dev/cobblesword/nachospigot/CC.java

```diff
@@ -1,86 +0,0 @@
-package dev.cobblesword.nachospigot;
-
-import org.bukkit.ChatColor;
```

Comment:
Why remove this?

Reply:
It's unused and can be easily substituted with Bukkit's ChatColor.

Reply:
CC is easier to read and write, eg
ChatColor.BOLD.toString() + ChatColor.BLUE + "My Message" vs CC.bBlue

Reply:
> CC is easier to read and write, eg ChatColor.BOLD.toString() + ChatColor.BLUE + "My Message" vs CC.bBlue

Either way, this isn't used in the code and plugins will most likely not use it to keep compatibility with other server softwares and versions.